app/Calculator.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable require-jsdoc */
+class Calculator {
+  constructor() {}
+
+  add(arg1, arg2) {
+    if (typeof arg1 !== 'number' || typeof arg2 !== 'number') {
+      throw new Error(`only numbers can be entered`);
```

Comment:
Quote style should correspond to unified style of the whole project. Please check all project js-files.